Dockerfile

```
@@ -1,6 +1,6 @@
-FROM python:3.14.7
-# ghapi 2.x went async and this script didn't.
-RUN pip install 'ghapi<2'
+FROM python:3.14.6
+# sync=True and sync_paged only landed in 2.x.
+RUN pip install 'ghapi>=2'
```

Hadolint (2.15.1)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)